website/pages/en/index.js

```diff
@@ -429,7 +505,12 @@ const IndexContainer = styled.div`
   .grey-section {
     background-color: #f9fafc;
   }
-  h1, h2, p, span, a, button {
+  h1,
+  h2,
+  p,
+  span,
+  a,
+  button {
     font-family: 'HankenGroteskRegular', 'Overpass', sans-serif !important;
```

Comment:
This `!important` is the reason why you need `!important` elsewhere to overwrite the `font-family` right? Is it necessary?

Reply:
You are right, it is no necessary :)

Reply:
Awesome, so probably you won't need `!important` elsewhere 😄